CHANGELOG.md

```diff
@@ -49,7 +49,7 @@ Note that this project **does not** adhere to [Semantic Versioning](http://semve
 - We fixed issues with save actions not correctly loaded when opening the library. [#9122](https://github.com/JabRef/jabref/pull/9122)
 - We fixed an issue where title case didn't capitalize words after en-dash characters. [#9068](https://github.com/JabRef/jabref/pull/9068)
 - We fixed an issue where JabRef would not exit when a connection to a LibreOffice document was established previously and the document is still open. [#9075](https://github.com/JabRef/jabref/issues/9075)
-
+  We fixed an issue where title case was capitalizing Conjunctions after en-dash characters and the Capitalization fix made as part of #9012 has been restricted only to Title case. #9112
```

Comment:
Actually in this special case you don't need a changelog entry, as the current version of JabRef is still unreleased and the error was introduced after the last release.

For nearly all other cases, a changelog entry is necesary.

Reply:
Sure will remove the new line.Should I modify the existing changelog entry made as part of the #9102

Reply:
yes I think that's a good idea. Ref this issue/pr then as well

Reply:
Modified the same line and included this PR details to it.